tests/framework/installer/ceph_manifests.go

```diff
@@ -97,86 +97,55 @@ func NewCephManifests(version string) CephManifests {
 
 func (m *CephManifestsMaster) GetRookCRDs() string {
 	return `
-apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
The tests need to keep both the v1beta1 and v1 CRDs available to test all the supported versions of K8s. If it's k8s 1.15 or older, we need to use the v1beta1, and for newer we use the v1.  Notice the CI is failing on 1.15 due to this.

Reply:
Done.